index.html

```diff
@@ -140,13 +140,12 @@
             </div>
             <div class="tile is-child is-12 notification is-primary"> <!-- First Column About Results Square -->
                 <p class="title">About</p>
-                <p class="subtitle">Language:</p>
-                <p class="subtitle">Population:</p>
-                <p class="subtitle">Time Zone:</p>
-                <p class="subtitle">Currency:</p>
-                <div class="columns">                                       <!--ALERT! Need to adjust alignment for Flag and img element. Too much spacing-->
+                <p class="subtitle" id="lang1">Language:</p>
+                <p class="subtitle" id="pop1">Population:</p>
+                <p class="subtitle" id="currency1">Currency:</p>
```

Comment:
Adding IDs was good addition to each p tag. Please leave comments to update other devs should be wary of adjustments or if it was fixed.